index.js

```diff
@@ -36,7 +36,7 @@ program
     co(function * () {
 
       try {
-        fs.accessSync( Path.resolve(process.cwd(), `..`, `modules/${module}/config.js` ) )
+        fs.accessSync( Path.resolve(__dirname,`modules/${module}/config.js` ) )
```

Comment:
This needs to stay `process.cwd()`, which gets the current working directory of where you're running the command. `__dirname` will point to `pult`'s own folder within `node_modules/`